### PR TITLE
React week1 HW Todo list

### PR DESCRIPTION
**Pull request by Hani-far**

### CODE REVIEW

File: react/week1/todo/src/toDoList.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        <div>
+            <p>Deadline date: {props.date.day}/ {props.date.month}/ {props.date.year}</p>
+            <p>Status : {status}</p>
+        </div>
+    );
+}
+
+function ToDoItems(props) {
+    return (
+        <li>
```
Comment (senner008) [inline react/week1/todo/src/toDoList.jsx:16]: See if you can use a library to handle dates instead. for instance moment.js

File: react/week1/todo/src/toDoList.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    const status = deadline > today ? "valid" : "expired";
+    return (
+        <div>
+            <p>Deadline date: {props.date.day}/ {props.date.month}/ {props.date.year}</p>
+            <p>Status : {status}</p>
+        </div>
+    );
+}
+
+function ToDoItems(props) {
```
Comment (senner008) [inline react/week1/todo/src/toDoList.jsx]: I assume you mean singular TodoItem here.

**Review by senner008 — APPROVED** [review]
Good
A few less important comments

File: react/week1/todo/src/toDoList.jsx (hunk 2 above)
Comment (Hani-far) [reply react/week1/todo/src/toDoList.jsx]: yes, thanks

File: react/week1/todo/src/toDoList.jsx (hunk 1 above)
Comment (Hani-far) [reply react/week1/todo/src/toDoList.jsx:16]: sure, will try to do that. Thanks for your time and feedback

**Review by senner008 — APPROVED** [review]

File: react/week1/todo/src/toDoList.jsx (hunk 1 above)
Comment (Hani-far) [reply react/week1/todo/src/toDoList.jsx:16]: It's done, thank you so much

**Pull request merged**